packages/ecs-agent/ecs.config

```diff
@@ -1,2 +1,6 @@
 ECS_LOGFILE=/var/log/ecs/ecs-agent.log
 ECS_LOGLEVEL="{{settings.ecs.loglevel}}"
+{{#if settings.network.http-proxy}}
+HTTP_PROXY={{settings.network.http-proxy}}
+{{/if}}
+NO_PROXY={{#each settings.network.no-proxy}}{{#if @index}},{{/if}}{{this}}{{/each}}{{#if settings.network.no-proxy}},{{/if}}169.254.169.254,169.254.170.2,/var/run/docker.sock
```

Comment:
I suspect we will want a helper for this. When I used the built-in handlebars functionality to create a comma-delimited and quoted list, it was observed that a helper would be easier to understand.

Reply:
I thought about this for a while and I think implementing a custom helper to do this would probably bit more work than it's worth. To fulfil my desired use-case, the helper would also need to understand how to concatenate both strings and collections.

If the helper is just for generating comma-delimited list out of an array. Then essentially the only part we're replacing is `{{#each settings.network.no-proxy}}{{#if @index}},{{/if}}{{this}}{{/each}}`, which on its own doesn't seem too bad in terms of readability.

I'm willing to be convinced though. Maybe if there are more instances of us doing this, I would probably think a custom helper is necessary.